include/aspect/geometry_model/spherical_shell.h

```diff
@@ -95,6 +95,18 @@ namespace aspect
 
           /**
            * @copydoc Manifold::normal_vector()
+           *
+           * We fudge here, but for a good reason. What the function is supposed
+           * to compute is the normal vector to the surface. This *should* be the
+           * normal to the surface with topography, but instead we return the
+           * normal to the undeformed surface -- i.e., the radial direction. This
+           * is, in particular, used to compute no-flux boundary conditions,
+           * for which we want to impose something resembles a boundary
```

Comment:
```suggestion
           * for which we want to impose a boundary
```
Or at least I found the wording as it is right now confusing.